groovy/postInit/chemistry/organic_chemistry/petrochemistry/oil_refining/Hydrotreating.groovy

```diff
@@ -0,0 +1,53 @@
+import static globals.Petrochemistry.*
+
+DT = recipemap("sieve_distillation")
+FLUID_HEATER = recipemap("fluid_heater")
+FBR = recipemap("fixed_bed_reactor")
+BCR = recipemap("bubble_column_reactor")
+
+fractions.each { _, fraction ->
+    if (fraction.sulfuric) {
+        FLUID_HEATER.recipeBuilder()
+            .fluidInputs(fraction.getSulfuric(1000))
```

Comment:
this generated an error log, the cause is Sulfuric Vacuum Oil Residue is not registered yet